kubetest/main.go

```diff
@@ -495,7 +492,7 @@ func migrateGcpEnvAndOptions(o *options) error {
 			name:   "--gcp-network",
 		},
 		{
-			env:    nodeImage,
+			env:    "KUBE_NODE_OS_DISTRIBUTION",
 			option: &o.gcpNodeImage,
```

Comment:
Nope, looks like the flag clobbers the env: https://github.com/kubernetes/test-infra/blob/master/kubetest/util_test.go#L374